torchaudio/transforms.py

```diff
@@ -7,441 +7,362 @@
-
+            torch.Tensor: Scaled by the scale factor
         """
         return F.scale(tensor, self.factor)
 
-    def __repr__(self):
-        return self.__class__.__name__ + '()'
-
 
 class PadTrim(torch.jit.ScriptModule):
-    """Pad/Trim a 2d-Tensor (Signal or Labels)
+    r"""Pad/Trim a 2D tensor
 
     Args:
-        tensor (Tensor): Tensor of audio of size (n x c) or (c x n)
-        max_len (int): Length to which the tensor will be padded
-        channels_first (bool): Pad for channels first tensors.  Default: `True`
-
+        max_len (int): Length to which the waveform will be padded
+        fill_value (float): Value to fill in
     """
-    __constants__ = ['max_len', 'fill_value', 'len_dim', 'ch_dim']
+    __constants__ = ['max_len', 'fill_value']
 
-    def __init__(self, max_len, fill_value=0., channels_first=True):
+    def __init__(self, max_len, fill_value=0.):
         super(PadTrim, self).__init__()
         self.max_len = max_len
         self.fill_value = fill_value
-        self.len_dim, self.ch_dim = int(channels_first), int(not channels_first)
 
     @torch.jit.script_method
-    def forward(self, tensor):
-        """
+    def forward(self, waveform):
+        r"""
+        Args:
+            waveform (torch.Tensor): Tensor of audio of size (c, n)
 
         Returns:
-            Tensor: (c x n) or (n x c)
-
+            Tensor: Tensor of size (c, `max_len`)
         """
-        return F.pad_trim(tensor, self.ch_dim, self.max_len, self.len_dim, self.fill_value)
-
-    def __repr__(self):
-        return self.__class__.__name__ + '(max_len={0})'.format(self.max_len)
+        return F.pad_trim(waveform, self.max_len, self.fill_value)
 
 
 class DownmixMono(torch.jit.ScriptModule):
-    """Downmix any stereo signals to mono.  Consider using a `SoxEffectsChain` with
-       the `channels` effect instead of this transformation.
-
-    Inputs:
-        tensor (Tensor): Tensor of audio of size (c x n) or (n x c)
-        channels_first (bool): Downmix across channels dimension.  Default: `True`
-
-    Returns:
-        tensor (Tensor) (Samples x 1):
-
+    r"""Downmix stereo waveform to mono.  Consider using a `SoxEffectsChain` with
+    the `channels` effect instead of this transformation.
     """
-    __constants__ = ['ch_dim']
-
-    def __init__(self, channels_first=None):
+    def __init__(self):
         super(DownmixMono, self).__init__()
-        self.ch_dim = int(not channels_first)
 
     @torch.jit.script_method
-    def forward(self, tensor):
-        return F.downmix_mono(tensor, self.ch_dim)
+    def forward(self, waveform):
+        r"""
+        Args:
+            waveform (torch.Tensor): Tensor of audio of size (c, n)
 
-    def __repr__(self):
-        return self.__class__.__name__ + '()'
+        Returns:
+            torch.Tensor: Tensor that has been downmixed of size (1, n)
+        """
+        return F.downmix_mono(waveform)
 
 
 class LC2CL(torch.jit.ScriptModule):
```

Comment:
What about this: If we agree that all signal must be of form C x L and torch.load produces C x L then any user should be responsible for this conversion herself? - Similar to scale, this wraps a simple function 'transpose'.

Reply:
Done.